cpp/src/randomforest/randomforest.cu

```diff
@@ -259,6 +259,23 @@ void _print_rf(const RandomForestMetaData<T, L>* forest, bool summary) {
   }
 }
 
+template <class T, class L>
+std::string _dump_rf_as_json(const RandomForestMetaData<T, L>* forest) {
+  if (!forest || !forest->trees) {
+    return "[]";
```

Comment:
Should we also dump tree metadata somewhere? Like: classification/regression, n_estimators, all other parameters passed to __init__

Reply:
Not sure that's necessary, since we are not implementing a full serialization format but a human-readable dump.

Reply:
I'm ok holding this off a bit until a future PR, but I do think that dumping the metadata will be very helpful. It doesn't have to be for full serialization, but when comparing trees, visualizing, etc., it'll be useful to have the metadata handy.